src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -61,12 +69,34 @@ EffectManifestPointer PitchShiftEffect::getManifest() {
     pitch->setNeutralPointOnScale(0.0);
     pitch->setRange(-1.0, 0.0, 1.0);
 
+    EffectManifestParameterPointer semitonesMode = pManifest->addParameter();
+    semitonesMode->setId("semitonesMode");
+    semitonesMode->setName(QObject::tr("Semitones"));
+    semitonesMode->setShortName(QObject::tr("Semitones"));
+    semitonesMode->setDescription(QObject::tr(
+            "Change pitch in semitone-steps instead of smoothly."));
```

Comment:
I purposefully suggested "smooth" instead of "continuous" because these texts should be as easy to understand as possible. IMO knowing the word "continuous" requires a higher level of english skills than "smooth". Even though "continuous" would probably be the better antonym.

Reply:
> Sounds more natural for me, not sure though ...
You could have even a smooth change to full semitone steps, which is a different feature.

I agree, based on some grammar upgrades, the `"Change the pitch in semitone steps instead of continuously."` version looks good to me.

Reply:
@Swiftb0y sorry, based my browser did not update your answer, I see it until now. IMHO the "continuously" looks more accurately in some way to me. To go a little bit deeper in the English point of view, both words belong to the B2 level, but it's true that the "continuously" word is rather for more technical English users. Of course, I am open-minded to both versions.

Reply:
Since this is a translated text, there is no much need to be understandable for non English speaker. As long it is correct English it works for me. 

It just jumps to my eyes initially because It look unusual to me. But I can*t really judge. Maybe a native speaker can jump in, @ywwg? This is not a blocking issue, though.